DataAccess/DA_Users.cs

```diff
@@ -1,35 +1,60 @@
 ﻿using System;
+using System.Linq;
 using System.Collections.Generic;
+using System.Text;
 using System.Data;
 using Models;
-using PagedList;
 
 namespace DataAccess
-{    
-   public class DA_Users :Utility
-    {      
-        List<Users> listuser = null;
-        /// <summary>
-        /// Description:To get the active user list from database
-        /// Name:Bhargav krishna
-        /// </summary>
-        
-        public List<Users> GetUsers()
```

Comment:
Why the existing method is removed from code and is it still available under any other class. if so provide details.